src/components/stepper/Stepper.tsx

```diff
@@ -0,0 +1,67 @@
+import StepperCircle from "./StepperCircle"
+import { MainContainer } from "./Stepper.styles";
+import { createRef, useRef, useState } from "react";
+import { Button, ButtonsContainer, StepperContainer, FormContainer } from "./StepperHandler.styles";
+
+type Step = {
+    title: string;
+    component: React.ForwardRefExoticComponent<any>;
+    props?: any;
+}
+
+
+interface StepperProps {
+    steps: Step[];
+}
+
+
+function Stepper({ steps }: StepperProps) {
```

Comment:
Acá deberíamos agregar el onCancel como parametro


```suggestion
interface StepperProps {
    steps: Step[];
    onCancel: () => void;
}


function Stepper({ steps, onCancel }: StepperProps) {
```